Resources/Prototypes/Entities/Objects/Specific/chemical-containers.yml

```
@@ -538,4 +540,18 @@
       beaker:
         reagents:
         - ReagentId: WeldingFuel
+          Quantity: 200
+- type: entity
+  parent: Jug
+  suffix: holy water
+  id: Jugholywater
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Label
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)